hail/python/hail/vds/sample_qc.py

```diff
@@ -0,0 +1,310 @@
+from typing import Optional, Sequence
+
+import hail as hl
+from hail.expr.expressions import Expression
+from hail.expr.expressions.typed_expressions import (
+    ArrayExpression,
+    CallExpression,
+    LocusExpression,
+    NumericExpression,
+    StructExpression,
+)
+from hail.methods.misc import require_first_key_field_locus
+from hail.table import Table
+from hail.typecheck import sequenceof, typecheck, nullable
+from hail.utils.java import Env
+from hail.utils.misc import divide_null
+from hail.vds.variant_dataset import VariantDataset
+
+
+@typecheck(global_gt=Expression, alleles=ArrayExpression)
+def vmt_sample_qc_variant_annotations(
```

Comment:
For both expert users and ourselves, can you add some minimal documentation for these auxiliary methods? I don't know if they should be linked in the docs or not, but they should at least have docstrings.